packages/mui-joy/src/styles/extendTheme.ts

```diff
@@ -623,5 +625,10 @@ export default function extendTheme(themeOptions?: CssVarsThemeOptions): Theme {
     attachColorChannels(colorSystem.palette);
   });
 
+  theme.sxConfig = {
```

Comment:
To start with an unstable API, assuming we want to make it eventually stable? 

```suggestion
  theme.unstable_sxConfig = {
```

Reply:
Stitches would call it

```suggestion
  theme.utils = {
```

https://stitches.dev/docs/utils 

but I like sxConfig better.

In sx: Does the s mean style or system or else? Does the x mean expand or expression or else?

Reply:
`sx` sounds a lot better than utils.

> In sx: Does the s mean style or system or else? Does the x mean expand or expression or else?

Honestly, I don't really care about the meaning or abbreviation of it (we might not need to find a name for it 😅). What I care about as a consumer is what it does since the name `sx` is pretty unique.

Reply:
sx = style extension, the same as the style prop, but better :))

Reply:
> sx = style extension

ok sold